jbmc/src/java_bytecode/java_static_initializers.h

```diff
@@ -48,6 +48,31 @@ code_ifthenelset get_clinit_wrapper_body(
   const select_pointer_typet &pointer_type_selector,
   message_handlert &message_handler);
 
+/// Creates the body of a json_clinit function, which includes assignments for
+/// all static fields of a class to values read from a JSON file. If the JSON
```

Comment:
:bulb: Maybe clarify that the `class` we're talking about here is the one where `function_id` is declared (if I understood the code correctly). By the way, it's not clear why we pass a function. We should rather pass the class directly.

Reply:
Thanks, the `function_id` was legacy stuff - replaced it with `class_id`.